app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_many :expenses, foreign_key: :author_id
+  validates :name, presence: true
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] [OPTIONAL] I will like to recommend that you enable `:confirmable` so that users will be prompted to confirm their passwords after signing up and before signing in. This will enable more security for your papp.